builtin/providers/aws/structure.go

```diff
@@ -622,12 +623,19 @@ func flattenOptions(list []*rds.Option) []map[string]interface{} {
 				r["db_security_group_memberships"] = dbs
 			}
 			if i.OptionSettings != nil {
+				spew.Sdump("Options HERE %s", i.OptionSettings)
 				settings := make([]map[string]interface{}, 0, len(i.OptionSettings))
 				for _, j := range i.OptionSettings {
-					settings = append(settings, map[string]interface{}{
-						"name":  *j.Name,
-						"value": *j.Value,
-					})
+					spew.Sdump("Option HERE %s", j)
```

Comment:
And that is the perfect reason why we send PRs for review :) Removed now so will merge on green